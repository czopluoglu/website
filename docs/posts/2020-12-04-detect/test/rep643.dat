11000101011111010110
00000100001011100110
10010111000000101011
10100001011101001101
11001011111101101111
10000111001110110110
11001110110101111101
11100001001000101011
11111111110010111101
10011111111111000100
01111111111101011111
00111101110000101110
10110101011000110110
00101001101000111001
10010101111110101100
00010000001010100010
10110001111100001100
01001011110010100011
01111110101111111111
10111111110101011111
10111110111111101101
00010011001000001111
10010001001000110011
01110010111110100011
01000000100001101101
00010000011000110010
01101111111000001010
11011000100000000000
00101011000011000011
01000010110000100011
01001011001111110111
11001110110110101011
00111111111110101111
10010001010010100010
10011111111100111010
00101001011010000001
01010100101110100111
11001101110000110010
00001010111010110100
11000000111001101110
10101110111110111111
00001001101100101010
01011101101111101111
01111001100010001100
01110101111111011111
11111001110110100010
10001000001100111111
00111101011010000111
11100000100010110111
01011001110010011110
11000101111000100010
10000001010001001111
10010000101100111011
10011101111010001110
11111111110010111110
01010110111101010010
00111111111101111011
11010101011001101011
11111100101000111011
11101001111111111111
10000011111111111110
11010001001010100011
01001101111000011111
10100111111010101110
00110011011101110101
00000001001000100110
00111101111111001111
11101000111011100111
00110111111101101011
00110001101000001011
11010110111010000111
01111111111111111111
11100001011001110110
01000100111110101011
11110001101011000010
11111001001010100010
10011100101011110000
11010000011100000111
01011111110110100111
11010111011111111111
11111101101010000111
00100001011001001101
00001000100010100010
11111111000000100110
10011111100010010110
10010001011010010011
01001001101010101111
01000011111001101000
01110110010000110010
10111111101111101110
01111001111000111110
10101101011100110111
01010101010000000001
11111110111010110010
00110011001111000010
11010111110111110111
00111001000000010110
10110111101000000010
00000000001000100001
01010101011000000010
10110100010011111011
11101000110010101010
00000000001000110111
00110001111010111110
11000110011010111101
11011111111101001111
11101111001011010111
11001001010100111001
10101101000101101111
01100000001110110111
00010100001000110111
00001000011110111111
10100111001110101001
00101001001000001010
11101110110010100110
00100101000110011011
11000010011100000010
00011001100010101110
00010011101001101000
00001100011011110111
00010100000100001101
11000001100011010010
01101001001100000011
10101000011100101111
01010111001100101110
11010001001101111010
11000001000000000011
10000000100111000111
01111111111110011111
10101111111011010010
11111111111111101100
10100100101110000011
01100111001011011110
11000010011001010110
11011100111011111011
11001000111110100110
11101111110000000010
10001001001110110110
10000101011000100101
11011011101110100010
10100010111100100001
10011001011101011111
01001011100010101111
11010001011010010000
01110111110111100100
01100100111010110111
11110101111111001110
11000100100111010110
00000111011101101111
00001111100001100110
00000101001011100110
00001000001100111111
11000101111111111111
10001000100110001010
00110101100110100110
00000110101110101111
00000001011111100111
11001001101010001010
10100111111011100111
00101111111000110000
10100100111110111111
10110101001101111101
00001101001110100010
00001101101111011111
11010001011011010111
11001111111010000010
01000011111011111111
10011111111000110110
10000111001111101110
00000010110100001101
00110000011001101010
11001101111100111111
11010100001111111000
11110011111110101000
00010111111011011001
00010100101000001010
01001000111000000100
00100001001010110111
11101101111101101111
10101111011101011101
10000001010101011111
00011111110101100011
01010001010000000100
10111111111100100011
11111010111010101111
00011101011010000010
11000111001011011110
00010000101010000010
00110111011111100111
11000101101011011011
01000001011111110111
11101011111010010011
01101110101100111111
00111101010000110100
11001011100010100110
11101111110111001000
00010101101100100110
01110101111100101010
00000011100010100110
01000000001010100100
11111111011111010011
11001011011011010111
10110000111011100111
00001000101101010111
10111111011100101110
00100101001010000110
01111011001000111111
11011010001100011110
10010000010010001100
11011101111100101111
11001011111101000111
11100100111110100011
01000001011010100110
01001001111000110111
11001101011111110110
00001001110100110110
11011001001000111001
10000000101101000111
01001101101101011110
01010101011111110111
00100101111000010110
00111010111110111110
01011000111110100111
11011111111101100111
11011111010011010100
11010011111110010111
11101101001010111111
00110111101110000100
11110001010000101011
00111011110111010110
10111000011111011111
01000010000010111111
00000100110001001110
00001111001010110001
11111111011101110011
10010101100001010010
10000001000000010011
01000001000010001111
01000000010101000000
01001100000000001011
10011001110010000110
11101001111000110010
10100100011000110010
00010011111110101111
10101011110110110100
11110111001011011111
11010111011010110110
10000100001000110110
10100011110110111011
11011000001011100111
