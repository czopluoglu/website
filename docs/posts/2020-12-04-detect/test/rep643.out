-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep643.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.00

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.00
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.8122
                   IDN index value:   0.7211
                           Ratio r:   0.6672

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    1    2    2    1    2    2    2    1    1
    3    3    4    3    4    3    3    3    4    3

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    2    5    9   10

   -----------CLUSTER  2-------------

    3    4    6    7    8

   -----------CLUSTER  3-------------

   11   12   14   16   17   18   20

   -----------CLUSTER  4-------------

   13   15   19
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+--+--+----++------
   +d+++---+----+--++-+
   -+d+-+--++---+-----+
   -++d+-+++--+--+--+-+
   ++-+d++-++--------+-
   --+-+d+++++---+-----
   ---+++d+++----------
   +--+-++d+++-++------
   -+++++++d+-++----+--
   --+-+++++d--+-+-----
   -----+-+--d+++--+-++
   ---+----+-+d+-+++-+-
   +------+++++d+++++--
   +++----+--+-+d-++--+
   ---+-+---+-++-d++-+-
   -----------++++d++-+
   -+--------++++++d-++
   -+-+----+---+--+-d-+
   ----+-----++--+-+-d+
   -+++------+--+-++++d
 
 No cross validation for this DETECT run
